src/wrappers/ThermalProbe_TC.cpp

```diff
@@ -127,11 +137,7 @@ void ThermalProbe_TC::setCorrection(float value) {
  */
 float ThermalProbe_TC::getSampleMean() {
   // This is not used for the Liquid Crystal display, so we don't restrict range to 0 to 99.99
-  if (sampleSize > 0) {
-    return (sumOfSampleValues / sampleSize) + correction;
-  } else {
-    return 0.0;
-  }
+  return myStats.average();
```

Comment:
I think this should be changed to `return myStats.average() + correction;`

Reply:
Done.